paparazzi-gradle-plugin/build.gradle

```diff
@@ -23,12 +23,17 @@ gradlePlugin {
 dependencies {
   compileOnly gradleApi()
   implementation platform(libs.kotlin.bom)
-  implementation libs.plugin.kotlin
-  implementation libs.plugin.android
-  implementation(libs.tools.sdkCommon) {
+  compileOnly libs.plugin.kotlin
+  compileOnly libs.plugin.android
+  compileOnly(libs.tools.sdkCommon) {
     because "SymbolUtils.getPackageNameFromManifest removed in AGP 7.0.  Replace?"
   }
 
+  testImplementation libs.plugin.kotlin
+  testImplementation libs.plugin.android
+  testImplementation(libs.tools.sdkCommon) {
```

Comment:
Were you trying to replace the `runtimeOnly` part of the `implementation` with these to help Gradle TestKit?

<details><summary>If so, I hacked around this area of Gradle before, and got it working.</summary>

A likely cause for the [CI failure](https://github.com/cashapp/paparazzi/actions/runs/6278471146/job/17052299239?pr=1105#step:5:17129) is `PluginUnderTestMetadata.getPluginClasspath`, because the [default only uses production runtime classpath](https://github.com/gradle/gradle/blob/v8.3.0/subprojects/plugin-development/src/main/java/org/gradle/plugin/devel/plugins/JavaGradlePluginPlugin.java#L203).

Here's my solution: [definition](https://github.com/TWiStErRob/net.twisterrob.gradle/blob/e132379a7b51ad0ac2bef5e7f7df3767fca7fb92/gradle/plugins/src/main/kotlin/net/twisterrob/gradle/build/testing/net.twisterrob.gradle.build.testing.plugin-under-test-metadata-extras.gradle.kts#L64-L70) and [usage](https://github.com/TWiStErRob/net.twisterrob.gradle/blob/e132379a7b51ad0ac2bef5e7f7df3767fca7fb92/plugin/building/build.gradle.kts#L34-L39).  
(If you use this, you could probably ignore my version constraint usage, I needed it because I have AGP/KGP in the test matrix.)

</details>

Reply:
The above should work more or less without any hacks, not sure why CI is failing. WIll look more later